react/features/invite/components/InviteButton.web.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Button from '@atlaskit/button';
+import DropdownMenu from '@atlaskit/dropdown-menu';
+
+import { translate } from '../../base/i18n';
+import { getLocalParticipant, PARTICIPANT_ROLE } from '../../base/participants';
+
+import { openDialog } from '../../base/dialog';
+import { AddPeopleDialog, InviteDialog } from '.';
+import { DialOutDialog } from '../../dial-out';
+
+declare var interfaceConfig: Object;
+
+const DIAL_OUT_NAME = 'dialout';
+
+/**
+ * The button that provides different invite options.
+ */
+class InviteButton extends Component {
+
+    /**
+     * {@code InviteButton}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Indicates if the "Add to call" feature is available.
+         */
+        _isAddToCallAvailable: React.PropTypes.bool,
+
+        /**
+         * Indicates if the "Dial out" feature is available.
+         */
+        _isDialOutAvailable: React.PropTypes.bool,
+
+        /**
+         * The function closing the dialog.
+         */
+        openDialog: React.PropTypes.func,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func
```

Comment:
I saw in another PR the pattern of storing PropTypes in an intermediary variable. I think that should be standard now because React.PropTypes is being deprecated.

Reply:
I saw we already import prop-types in AbstractApp.js, so I'll start to use this one directly.